content/user/ui/report/Vulnerability-trends.md

```diff
@@ -0,0 +1,67 @@
+<!--
+title: "Vulnerability Trends"
+description: "Overview of vulnerability trend reports"
+tags: "TeamServer application vulnerability PDF report"
+-->
+ 
+Vulnerability management is a vital responsibility of any security team. Use the Vulnerability Trend reports to recognize the vulnerabilities your applications face and how well they're being managed so that you have a better understanding of your security posture.
+
+## Access Data
+ 
+Select **Reports** in the User menu to go the Vulnerability Trend dashboard. Click the **View** link to see the graphs in more detail. 
+
+Select **New** to see a graph of new vulnerabilities. Select **Total** to see a graph of all reported vulnerabilities compared to all remediated vulnerabilities. Each black data point in the graph generates a tooltip with the number of Suspicious, Confirmed and Reported vulnerabilities on that date when you hover over it. Each green data point generates a tooltip with the number of Not A Problem, Remediated and Fixed vulnerabilities on that date.
```

Comment:
Can we just say something like "Each black data point represents the total number of Suspicious, Confirmed and Reported vulnerabilities for that date. Green data points represent the total number of vulnerabilities marked as Not A Problem, Remediated or Fixed. Hovering over each data point will generate a tooltip with those status breakdowns."